sale_lead_time_range/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2024 ForgeFlow S.L. (http://www.forgeflow.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+{
+    "name": "Lead Time Range - Sale",
+    "version": "15.0.1.0.1",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/sale-workflow",
+    "category": "Sales",
+    "license": "AGPL-3",
+    "depends": ["sale_order_line_menu", "stock"],
```

Comment:
why is needed to depend on sale_order_line_menu?

Reply:
You are right, it should be sale instead of sale_order_line_menu